c8ylp/cli/core.py

```diff
@@ -629,32 +630,40 @@ def start_proxy(
         "ping_interval": opts.ping_interval,
     }
 
-    tcp_server = None
+    server = None
     background = None
 
     try:
-        tcp_server = TCPProxyServer(
-            opts.port,
-            WebsocketClient(**client_opts),
-            opts.tcp_size,
-            opts.tcp_timeout,
-        )
+        if opts.socket_path is not None:
+            server = UnixStreamProxyServer(
```

Comment:
This will not work in non-unix OS. We should handle this and don't allow "--socket-path" on non-unix OS

Reply:
renamed it to socket_server